packages/flutter_test/lib/src/recording_canvas.dart

```diff
@@ -194,6 +195,7 @@ class TestRecordingPaintingContext extends ClipContext implements PaintingContex
     Offset offset, {
     Rect? childPaintBounds,
   }) {
+    _layersToDispose.add(childLayer);
```

Comment:
same here, i think.

Reply:
Yes, I think this one makes it: https://github.com/flutter/flutter/pull/135942